templates/user/settings/appearance.tmpl

```diff
@@ -70,101 +70,106 @@
 		</div>
 
 		<!-- Shown comment event types -->
-		<h4 class="ui top attached header">
-			{{.locale.Tr "settings.hidden_comment_types"}}
-		</h4>
-		<div class="ui attached segment">
-			<form class="ui form" action="{{.Link}}/hidden_comments" method="post">
-				{{.CsrfTokenHtml}}
-				<div class="inline field">
-					<div class="ui checkbox">
-						<input name="reference" type="checkbox" {{if(call .IsCommentTypeGroupChecked "reference")}}checked{{end}}>
-						<label>{{.locale.Tr "settings.comment_type_group_reference"}}</label>
+		<div class="ui header">
```

Comment:
Does this new `<div class="ui header">` have a special purpose?

There is already `<h4 class="ui top attached header">` below.

And I do not think put `<div class="ui attached segment">` inside a `header` is correct layout.

ps: Fomantic UI has its best practices, I guess many UI problems are caused by misusage on Gitea side, Fomantic UI is innocent in many cases 🤣